### PR TITLE
add uninsert method

### PR DESCRIPTION
**Pull request by jnsbck**
This PR adds an `uninsert` method. 

I find the name a bit wonky tbh. Was also wondering if we should add a `disconnect` method to `connect.py`.

todos:
- [x] add tests

### CODE REVIEW

**Comment by michaeldeistler** [comment]
Regarding the name: I am not a huge fan of `uninsert` either, but not sure about other options. Maybe we could go with `delete_channel()` (similar to `delete_recordings()`)?

**Comment by jnsbck** [comment]
I like delete_channel much better! @michaeldeistler if you could have a quick look, that'd be great :)

File: jaxley/modules/base.py [hunk 1]
```diff
@@ -1735,6 +1735,28 @@ def insert(self, channel: Channel):
...
+        name = channel._name
+        channel_names = [c._name for c in self.channels]
+        all_channel_names = [c._name for c in self.base.channels]
+        if name in channel_names:
+            channel_cols = list(channel.channel_params.keys())
+            channel_cols += list(channel.channel_states.keys())
+            self.base.nodes.loc[self._nodes_in_view, channel_cols] = float("nan")
+            self.base.nodes.loc[self._nodes_in_view, name] = False
+
+            # only delete cols if no other comps in the module have the same channel
```
Comment (michaeldeistler) [inline jaxley/modules/base.py:1752]: nice!

**Review by michaeldeistler — APPROVED** [review]
Amazing, thanks!

**Pull request merged**